version/news/NEWS-2023.12.0-ocean-storm.md

```diff
@@ -19,6 +19,7 @@
 - Better error message when user preferences fail to save due to folder permissions. (#12974)
 - RStudio now supports pasting of file paths for files copied to the clipboard. (#4572)
 - RStudio now supports duplicate connection names for Posit drivers. (rstudio-pro#5437)
+- Added Databricks integration to RStudio (Pro) which allows users to seamlessly interact with Databricks clusters and resources (rstudio-pro#4927)
```

Comment:
```suggestion
- Added Databricks integration to RStudio Pro Sessions which allows users to seamlessly interact with Databricks clusters and resources (rstudio-pro#4927)
```
I think "RStudio Pro Sessions" is the correct terminology, and we should move this to the Workbench section since those sessions are only available in Workbench. Unless this is referring to RDP, in which case we should refer to it as such.